bin/nrnivmodl_makefile_cmake.in

```diff
@@ -24,6 +24,9 @@ NRNLIB_FLAGS = -L$(libdir) -lnrniv
 OS_NAME := $(shell uname)
 _cm =,
 
+# CoreNEURON link flags
```

Comment:
Could we embed this into `ifeq ($(LinkCoreNEURON), true)`, would be faster to read the makefile ?